### PR TITLE
New Integration: OpenInterpreter

### PR DESCRIPTION
**Pull request by dot-Justin**
`integrations/openinterpreter.py`:
 - New Integration! #9 

`utils/llm_parse.py`:
  - Edited prompt to allow OI int
  - added parsing for OI

`config.json`:
  - Add OI config options

`requirements.txt`:
  - Include OI library for new int
 
`README.md`
  - Add OI to integrations list

`main.py`:
  - Minor wording changes

```
 (\   /)

(◠ + ◠ )
   ►◄
```

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -41,6 +41,16 @@
...
 	"lamathome_isenabled": true,
 		"lamathometerminate_isenabled": true,
+	
+	"openinterpreter_isenabled": true,
+		"openinterpreter_auto_run_isenabled": true,
+		"openinterpreter_verbose_mode_isenabled": "false",
+			"openinterpreter_verbose_mode_isenabled_comment": "Valid entries: 'true', 'false'. Must be a string for parsing in openinterpreter.py",
+		"openinterpreter_llm_api_base": "groq",
+			"openinterpreter_llm_api_base_comment": "Valid entries: 'groq', 'openai', your own api base url",
+		"openinterpreter_llm_api_key": "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",
```
Comment (pnivek) [inline config.json:51]: umm......sir....

File: config.json [hunk 2]
```diff
@@ -41,6 +41,16 @@
...
+		"openinterpreter_llm_model": "llama3-8b-8192",
```
Comment (pnivek) [inline config.json:52]: 8b default for open interpreter?

Comment (dot-Justin) [reply config.json]: oops. token revoked 👍

File: config.json (hunk 1 above)
Comment (dot-Justin) [reply config.json]: oops. token revoked 👍

File: config.json (hunk 2 above)
Comment (dot-Justin) [reply config.json]: changed sir... sorry sir...

**Review by pnivek — APPROVED** [review]
LGTM!

**Pull request merged**